blazor/diagram/tool-tip.md

```diff
@@ -19,4 +19,656 @@ By default, the diagram displays a tooltip that provides information about the s
+            {
+               Open = new TooltipAnimationSettings(){Effect = Effect.None},
+                Close = new TooltipAnimationSettings(){Effect = Effect.None}
+            }},
+            Constraints = NodeConstraints.Default|NodeConstraints.Tooltip,
+        };
+        nodes.Add(node);
+    }
+    //Change Animation at run time
+    private void AnimationChange()
+    {
+        nodes[0].Tooltip.AnimationSettings = new AnimationModel()
+            {
+                Open = new TooltipAnimationSettings() { Effect = Effect.FadeZoomIn, Duration = 100 },
+                Close = new TooltipAnimationSettings() { Effect = Effect.FadeZoomOut, Duration = 50 }
+            };
+    }
+}
+```
+
+The following code example illustrates how to set animation to the tooltip for connectors.
+
+```cshtml
+@using Syncfusion.Blazor.Diagram
+@using Syncfusion.Blazor.Popups
+<input type="button" value="Connector Animation" @onclick="AnimationChange" />
+<SfDiagramComponent Width="1000px" Height="500px" Connectors="connector" />
+@code
+{
+    //Defines diagram's connector collection.
+    DiagramObjectCollection<Connector> connector;
+
+    protected override void OnInitialized()
+    {
+        connector = new DiagramObjectCollection<Connector>();
+        Connector connectors = new Connector()
+        {
+            ID = "Connector1",
+            SourcePoint = new DiagramPoint() { X = 500, Y = 500 },
+            TargetPoint = new DiagramPoint() { X = 600, Y = 400 },
+            Tooltip = new DiagramTooltip(){Content="ConnectorTooltip",AnimationSettings = new AnimationModel()
+            {
+               Open = new TooltipAnimationSettings(){Effect = Effect.None},
+                Close = new TooltipAnimationSettings(){Effect = Effect.None}
+            }},
+            Constraints =  ConnectorConstraints.Default|ConnectorConstraints.Tooltip ,
+        };
+        connector.Add(connectors);
+    }
+    //Change Animation at run time
+    private void AnimationChange()
+    {
+        connector[0].Tooltip.AnimationSettings = new AnimationModel()
+            {
+                Open = new TooltipAnimationSettings() { Effect = Effect.ZoomIn, Duration = 500 },
+                Close = new TooltipAnimationSettings() { Effect = Effect.ZoomOut, Duration = 500 }
+            };
+    }
+}
+```
+
+## Tooltip Open Mode
+
+The mode on which the Tooltip is to be opened on a page, i.e., on hovering, focusing, or clicking on the target elements can be decided. On initializing the Tooltip, set the [OpensOn](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Diagram.DiagramTooltip.html#Syncfusion_Blazor_Diagram_DiagramTooltip_OpensOn) property with any one of the following values:
+
+* `Auto`
+* `Hover`
+* `Custom`
+* `Click`
+
+
+> By default, Tooltip is open mode is `Auto`.
+
+The following code example illustrates how to set open mode to the tooltip for nodes.
```

Comment:
The following code example illustrates how to set the open mode to the tooltip for the nodes.